examples/test/videosphere/index.html

```diff
@@ -9,15 +9,41 @@
   <body>
     <a-scene stats>
       <a-assets>
-        <video id="lego" preload="true"
-               src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/t-197/walking_the_dog_360_video_short.webm"
+        <!-- Test anonymous video with source element. -->
+        <!-- Normally, one would give the video an id, e.g. id="lego" -->
+        <video preload="auto"
                width="360" height="360" autoplay loop="true"
-               crossOrigin="anonymous"></video>
+               crossOrigin="anonymous">
+          <source src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/t-197/walking_the_dog_360_video_short.webm"></source>
+        </video>
       </a-assets>
-
-      <a-entity material="shader: flat; src: #lego" geometry="primitive: sphere; radius: 100"
+      <!-- Normally, one would use the video id, e.g. material="src:#lego" -->
+      <a-entity material="shader: flat" geometry="primitive: sphere; radius: 100"
                 scale="-1 1 1">
       </a-entity>
     </a-scene>
+
+    <!-- Normally, one would use video id, e.g. material="src:#lego" above. 
```

Comment:
this is a repeat of the comment on [line 20](https://github.com/aframevr/aframe/pull/3176/files#diff-a9868d4845b66210ef74e2f22a86c1fbR20). I'd remove this sentence.